.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 /docs/variables.mk.local
 /node_modules/
+/vale/Google/
```

Comment:
Could we keep this directory in the repository, and add the Google style guide files in this PR? I think this would make it easier for people to contribute to the repository and stick to the Vale guidelines without having to remember to run `vale sync`.

Reply:
Yeah perhaps going back to vendoring is the way to go. I'll add that back in and then ensure that the container image we build uses the vendored style.

Reply:
Done in [75c6f09](https://github.com/grafana/writers-toolkit/pull/292/commits/75c6f09d42ae15b732e779f355202197389f4f99).

FYI, I squashed the commits locally so you might need to reset your local branch if you have one.